package.json

```diff
@@ -17,43 +21,8 @@
     "lint:fix": "npm run lint -- --fix",
     "pre-commit": "remove-lockfiles && node index.js",
     "pretest": "npm run lint",
-    "test": "jest --coverage",
-    "deps": "npm-check -s",
-    "deps:update": "npm-check -u"
-  },
-  "pre-commit": "pre-commit",
-  "jest": {
-    "testEnvironment": "node",
-    "setupFiles": ["./testSetup.js"]
-  },
-  "greenkeeper": {
-    "ignore": [
-      "cosmiconfig"
-    ]
+    "test": "jest --coverage"
```

Comment:
Could we add a script for test+watch? Specifically, `"test:watch": "jest --watch"`

Reply:
is it really needed? we can just run "npm test -- --watch", right? or do you have a specific use case?

Reply:
Couple of reasons. It executes the lint script(`pretest`). It includes the coverage flag which is more work we don't need.

Reply:
Personally I prefer modifying our npm scripts to make them neater and less bloated
```diff
{
    "commit": "git-cz",
    "lint": "eslint .",
-   "lint:fix": "npm run lint -- --fix",
    "pre-commit": "remove-lockfiles && node index.js",
-   "pretest": "npm run lint",
-   "test": "jest --coverage",
+   "test": "jest"
}
```

Then update our `.travis.yml` to run these lines as `script` job
```yml
script:
  - npm run lint
  - npm run test -- --coverage
```

because running jest in watch mode or having jest to collect code coverage is just as easy as passing additional flags (--watch & --coverage)

Reply:
but if we all agree to add `"test:watch": "jest --watch"`
i will suggest to also add `"test:coverage": "jest --coverage"`

what do you think?

Reply:
All fine with me...

Reply:
One very small advantage of having specific scripts configured is being able to use autocomplete. I am okay with either of the suggested changes though.